dll/shellext/zipfldr/resource.h

```diff
@@ -5,8 +5,11 @@
 #define IDB_HEADER                     200
 #define IDB_WATERMARK                  201
 
+/* Icons */
+#define IDI_ZIPFLDR                    202
```

Comment:
Is it possible to use the original value of "1" ?

Reply:
Tried ID = 1, it works so i think it's OK.